client/extensions/woocommerce/state/ui/products/variations/test/edits-reducer.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import reducer from '../edits-reducer';
+
+import {
+	editProductVariation,
+} from '../actions';
+
+describe( 'edits-reducer', () => {
+	it( 'should initialize to null', () => {
+		expect( reducer( undefined, { type: '@@test/INIT' } ) ).to.equal( null );
```

Comment:
It's a throw-away type. It just needs to not be something that our reducer would catch.